src/tests/ilverify/ILTests/FieldTests.il

```diff
@@ -43,7 +43,7 @@
         ret
     }
 
-    .method public instance void Ldflda.InitonlyFieldOutsideCtor_Invalid_InitOnly() cil managed
+    .method public instance void Ldflda.InitonlyFieldOutsideCtor_Valid() cil managed
```

Comment:
Maybe add a link to the issue on verification of readonly references here as well.

Reply:
In the proposed design, it does not explicitly call out anything has changed for instance constructor to allow `ldflda` with readonly fields. My understanding from https://github.com/dotnet/roslyn/issues/22485 was that it was allowed before the proposed changes and it was just a bug in peverify that ilverify has inherited.

Reply:
ldflda was disallowed on initonly fields originally. From ECMA-335 - ldflda chapter: For verifiable code, field cannot be init-only.

The current ilverify version relaxed this to allow ldflda on initonly fields in constructors.

https://github.com/dotnet/designs/blob/main/proposed/verifiable-ref-readonly.md relaxes it even further and allows ldflda on initonly fields everywhere, but disallows the result of such operation to be written to, except in constructors.

Reply:
Added the comment.